grpc/sa-wrappers.go

```diff
@@ -283,6 +283,20 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	return sac.inner.AddPrecertificate(ctx, req)
+}
+
+func (sac StorageAuthorityClientWrapper) AddSerial(
+	ctx context.Context,
+	req *sapb.AddSerialRequest,
+) (*corepb.Empty, error) {
+	return sac.inner.AddSerial(ctx, req)
```

Comment:
Similar to `AddPrecertificate` I think this wrapper needs to check that `req` is `!= nil` and return an error if it is. The `sac.inner.AddSerial` implementation is de-referencing `req` without checking that.